TSPL.docc/LanguageGuide/Macros.md

```diff
@@ -230,7 +230,7 @@ adds new members to the type you apply it to.
 The `@OptionSet` macro adds an `init(rawValue:)` initializer
 that's required by the `OptionSet` protocol,
 as well as some additional members.
-The second use, `@attached(conformance)`, tells you that `@OptionSet`
+The second use, `@attached(extension)`, tells you that `@OptionSet`
 adds one or more protocol conformances.
```

Comment:
Similarly, this text should mention that the `@attached(extension)` macro also communicates which protocols the macro adds a conformance to.

Reply:
Fixed in d444c690.